test/provider-aws.tf

```diff
@@ -0,0 +1,22 @@
+variable "aws_assume_role_arn" {
+  description = "AWS role to assume for this run"
+}
+
+variable "aws_region" {
+  description = "GCP Region to use for this run"
+}
+
+provider "aws" {
+  dynamic "assume_role" {
+    for_each = var.aws_assume_role_arn != "" ? [] : [1]
+    content {
+      role_arn = var.aws_assume_role_arn
+      session_name = "terratest"
+    }
+  }
+
+  region = var.aws_region
+
+  # Jakarta not in provider whitelist at time of writing
```

Comment:
allowlist (as opposed to whitelist)

Reply:
Amusingly isn't in my IDE's dictionary yet.